app/views/episodes/_form_status.html.erb

```diff
@@ -54,7 +54,7 @@
               <ul class="dropdown-menu">
                 <% if episode.published? && !episode.published_at_changed? %>
                   <li><%= link_to t(".unpublish"), episode_path(episode, episode: {publishing_status: "draft"}), method: :put, class: "dropdown-item bg-danger text-white" %></li>
-                <% else %>
+                <% elsif policy(episode).destroy? %>
```

Comment:
This was giving unauthorized users a (non-functional) button to destroy episodes - whoops!